### ISSUE TITLE
Cannot install accessibility-developer-tools 

### ISSUE THREAD

**Issue by mateodelnorte** [npm/npm#15066]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [X] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

npm i accessibility-developer-tools fails with the following error: 
```
C:\development\electron-react-boilerplate>npm i accessibility-developer-tools
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files (x86)\\Nodist\\v-x64\\6.9.1\\node.exe" "C:\\Program Files (x86)\\Nodist\\npmv\\3.10.9\\bin\\npm-cli.js" "i" "accessibility-developer-tools"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.9

npm ERR! shasum check failed for C:\Users\Waltem7\AppData\Local\Temp\npm-6864-c93f02db\registry.npmjs.org\accessibility-developer-tools\-\accessibility-developer-tools-2.11.0.tgz
npm ERR! Expected: 2b669c802671672aed5c61c42a782082a2fc7d2c
npm ERR! Actual:   ba1966ded6484349ff3ed9f55708882ad5031690
npm ERR! From:     https://registry.npmjs.org/accessibility-developer-tools/-/accessibility-developer-tools-2.11.0.tgz
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\development\electron-react-boilerplate\npm-debug.log
```

### supporting information:

 - `npm -v` prints: 3.10.09
 - `node -v` prints: 6.9.1
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X/macOS, or Linux?: Windows
 - Network issues:
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [ ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

**Comment by Charlotteis** [comment]
Hey!

Does the problem persist if you update to the latest version of `npm`?

**Comment by mateodelnorte** [comment]
3.10.9 is latest, no?

**Comment by legodude17** [comment]
`4.0.2` is latest, and there is also 3.10.10 if you have to stay with npm@3.

**Comment by Charlotteis** [comment]
Could you try `npm cache clean`? :)

**Comment by zkat** [comment]
Closing as duplicate of https://github.com/npm/npm/issues/14626. See https://github.com/npm/npm/issues/14626#issuecomment-264371634 for some helpful steps.
